### PR TITLE
Update minor styling changes on land-use-action component

### PR DESCRIPTION
**Pull request by trbmcginnis**
Re-add styling changes that got lost in a past merge conflict for `landUseAction` component.

### CODE REVIEW

File: client/app/components/land-use-action.hbs [hunk 1]
```diff
@@ -29,17 +29,17 @@
...
+            <ActionFormInput 
+              @pasForm={{@pasForm}}
+              @fieldType="number"
+              @length="10"
+              @landUseActionField={{landUseAction.countField}}
+            />
+            <ValidationMessage
+              @changesetError={{get @submittableChanges.error landUseAction.countField}}
+            />
+          </div>
```
Comment (allthesignals) [inline client/app/components/land-use-action.hbs:42]: Hmm do you see the spacing issues in the diff here? It doesn't even look like anything was added. How come there are new spaces added?

Comment (trbmcginnis) [reply client/app/components/land-use-action.hbs:42]: The ActionFormInput was moved inside another div that was previously above it

Comment (trbmcginnis) [reply client/app/components/land-use-action.hbs:42]: It's hard to see on the git diff :/ 

Screen Shot 2020-05-19 at 12 15 41 PM

Comment (allthesignals) [reply client/app/components/land-use-action.hbs:42]: Oh sorry I think my Sourcegraph plugin is going haywire 
image

Comment (bfreeds) [reply client/app/components/land-use-action.hbs:42]: @allthesignals you can hide whitespace changes here:
image

**Pull request merged**